pkg/il/evaluator/evaluator.go

```diff
@@ -101,6 +101,12 @@ func (e *IL) AssertType(expr string, finder expr.AttributeDescriptorFinder, expe
 	return nil
 }
 
+// ChangeVocabulary handles changing of the attribute vocabulary.
+func (e *IL) ChangeVocabulary(finder expr.AttributeDescriptorFinder) {
+	e.finder = finder
```

Comment:
This seems non thread safe, in case the same IL is used across multiple request ? Or that is not the case ?

Reply:
That is correct, my email earlier in the day was about this issue.'

The function just below this ( `ConfigChange`),  works in the exact same way, and it is in use today. I think it works because  `df` the attribute finders do not change often (or at all).

I will fix it in a another PR.

Reply:
Yes..  let's not do in this PR.